ssn/integrated/ssn.ttl

```diff
@@ -37,7 +37,24 @@ vann:preferredNamespaceUri a owl:AnnotationProperty .
 ssn: a owl:Ontology ;
   owl:imports sosa: .
 
+## Features of interest and Property
 
+#sosa:FeatureOfInterest
+#  rdfs:subClassOf [ owl:onProperty ssn:hasProperty ; owl:someValuesFrom ssn:Property ] ; # TODO: need resolution, is it ssn:hasProperty, or sosa:hasProperty ?
+#  rdfs:subClassOf [ owl:onProperty ssn:hasProperty ; owl:allValuesFrom ssn:Property ]  . # TODO: need resolution, is it ssn:hasProperty, or sosa:hasProperty ?
+
+ssn:Property a owl:Class ;
+  rdfs:label "Property"@en ;
+  skos:definition """A Quality of a FeatureOfInterest. An aspect of an entity that is intrinsic to and cannot exist without the entity."""@en ;
+#  rdfs:subClassOf [ owl:onProperty sosa:isPropertyOf ; owl:someValuesFrom sosa:FeatureOfInterest ] ; # TODO: need resolution, is it ssn:isPropertyOf, or sosa:isPropertyOf ?
+  rdfs:isDefinedBy ssn: .
+
+  sosa:ObservableProperty rdfs:subClassOf ssn:Property .
+
+  sosa:ActuatableProperty rdfs:subClassOf ssn:Property .
+
```

Comment:
Implementation of the option 6 in https://www.w3.org/2015/spatial/wiki/Property_and_Observable_Property